Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/checkins/CheckInsViewModel.kt

```diff
@@ -24,8 +31,28 @@ class CheckInsViewModel @AssistedInject constructor(
     private val qrCodeUriParser: QRCodeUriParser
 ) : CWAViewModel(dispatcherProvider) {
 
-    private val verifyResultData = MutableLiveData<TraceLocationVerifyResult>()
-    val verifyResult: LiveData<TraceLocationVerifyResult> = verifyResultData
+    val confirmationEvent = SingleLiveEvent<TraceLocationVerifyResult>()
+
+    val checkins = FAKE_CHECKIN_SOURCE
+        .map { checkins -> checkins.sortedBy { it.checkInEnd } }
```

Comment:
Do we need 2nd level sort order - e.g. `checkInStart` to have a stable sorting for active events?

Reply:
Good point, by title would make sense, will add in follow up PRs.